content/posts/2023-03-05-gather-town-building.md

```diff
@@ -0,0 +1,186 @@
+Title: PyCon APAC 2022 on Gather.Town
+Date: 2023-03-05 21:00
+Modified: 2023-03-05 21:50
+Category: online-conference
+Tags: en
+Slug: gather-town-building
+Authors: Winnie, Fish, Ray, Shirley, Vivian, Pochun, TN-Lee, Yoyo, Angus, Allen
+Summary: Last year (2022), PyCon Taiwan hosted PyCon APAC for the third time. Due to global epidemic situation, we chose holding PyCon APAC 2022 remotely via Gather.Town. It’s the second time we used Gather.Town to build the conference. We found that Gather.Town kept updating the interface and features, and created lots of friendly example maps for newcomers. In this article, we would introduce the events of PyCon APAC 2022 in the following.  
+
+## Introduction 
+
+Last year (2022), PyCon Taiwan hosted PyCon APAC for the third time. 
+
+Due to global epidemic situation, we chose holding PyCon APAC 2022 remotely via Gather.Town. It’s the second time we used Gather.Town to build the conference. We found that Gather.Town kept updating the interface and features, and created lots of friendly example maps for newcomers.
+
+In this article, we would introduce the events of PyCon APAC 2022 in the following.   <i> - written by Winnie - </i>
+
+<!--more-->
+
+## Overall
+#### Space A & Space B 
+
+We tried to introduce Taiwan street view in our main spaces, i.e. Space A & Space B. There is a temple in a rice cooker shape. In Taiwan, there are more than 2,200 temples, which exceeds the number of the convenience stores.  <i> - written by Fish - </i>
+
+<!-- ![Rice Cooker Temple](https://i.imgur.com/skTuzqm.jpg) -->
+![Rice Cooker Temple](images/2023-03-05-gather-town-building/0.jpg)
```

Comment:
It'd be better if we could give a self explanatory name for the image as well.

Reply:
Sure.